wrangler-unenv-polyfills/wrangler.toml

```diff
@@ -1,4 +1,4 @@
 name = "unenv-worker"
 main = "worker.mjs"
-compatibility_flags = ["experimental:nodejs_compat_v2"]
-compatibility_date = "2024-08-29"
\ No newline at end of file
+compatibility_flags = ["nodejs_compat_v2", "nodejs_zlib"]
```

Comment:
we should also switch to `nodejs_compat` but we can't in this version of wrangler because of https://github.com/cloudflare/workers-sdk/issues/6821 but this is being fixed today, so maybe just add a TODO so that we fix it the next time we touch this file? thanks